src/main/java/christmas/domain/order/Order.java

```diff
@@ -0,0 +1,21 @@
+package christmas.domain.order;
+
+import christmas.constant.Constants;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public record Order(Map<MenuName, MenuCount> order) {
+
+    public long getTotalOrderCostBeforeDiscount() {
```

Comment:
메뉴는 최대 20개까지만 주문을 할 수 있기에 총 금액이 최대가 되는 경우 역시 `int`형으로 커버할 수 있는 범위라고 생각이 들어요!
```suggestion
    public int getTotalOrderCostBeforeDiscount() {
```